### DISCUSSION TITLE
Buena tarde, ayuda por favor al intentar hacer commit y hacer push mi cuenta dice que esta marcada como Spam como puedo solucionarlo ?

### DISCUSSION THREAD

**Discussion by jerson1946** [community/community#78689] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Buena tarde, ayuda por favor al intentar hacer commit y hacer push mi cuenta dice que esta marcada como Spam como puedo solucionarlo ?

**Comment by 0xlengo** [comment]
Hola [@jerson1946](https://github.com/jerson1946),
Gracias por compartir tu situación.

Cuando una cuenta de GitHub es marcada como spam, generalmente se debe a alguna actividad que los sistemas automatizados consideran sospechosa (por ejemplo, creación masiva de repositorios, uso de ciertas IPs o patrones de comportamiento).

Para resolverlo:

✅ Te recomiendo contactar directamente al equipo de soporte de GitHub mediante este formulario:
🔗 https://support.github.com/contact/account

Explicá tu situación y mencioná que estás teniendo problemas al hacer commit y push. Ellos podrán revisar tu cuenta y quitar el bloqueo si fue un error.

Mientras tanto, asegurate de:

Verificar tu dirección de correo electrónico en GitHub.

No usar VPNs o proxies que puedan generar sospechas.

No crear múltiples cuentas desde la misma red.

Espero que se solucione pronto. ¡Éxitos!
